### PR TITLE
Remove book a room link from website

### PR DESCRIPTION
**Pull request by jfilter**

### CODE REVIEW

File: README.md
Comment (riotbib) [inline README.md:1]: I'd recommend moving this to a separate PR.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # Cultivation Space
+
 The website for Cultivation Space in Berlin's Wedding at exRotaPrint
+
+## Installtion
```
Comment (riotbib) [inline README.md]: Small typo, missing `a`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+## Development
+
+```
+npm install
+hugo server
+```
+
+## Deployment
+
+Changes to the main branch will get automatically deployed.
```
Comment (riotbib) [inline README.md:18]: Are we sure about the automation part?

File: content/book.md
Comment (riotbib) [inline content/book.md:1]: Looks good to me!

**Review by riotbib — CHANGES_REQUESTED** [review]
Thanks a lot. Kinda wanted to do the same.

File: README.md (hunk 2 above)
Comment (jfilter) [reply README.md:18]: It was told so. We will find out. :)

File: README.md (hunk 1 above)
Comment (jfilter) [reply README.md]: Thanks

File: README.md
Comment (jfilter) [reply README.md:1]: I don't think this is necessary in this case.

**Comment by elioqoshi** [comment]
I think that would be good now that we have a forum. Ideally we should have a way for people to request a room even if they are not members (yet). A longer discussion though.
